.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": ["@babel/preset-env", ["@babel/preset-react", {
+     "runtime": "automatic"
+  }]]
+}
```

Comment:
Could you have all files end with a newline?

Reply:
Will double new lines after rebasing ESLint PR